docs/_documentations/remote-removing.md

```diff
@@ -21,29 +21,27 @@ There are two modes of use. You can:
 
 ## Removing a remote deployment of Codewind
 
-You will find `cwctl` in your HOME directory under the path ~/.codewind/{version}
+You can find `cwctl` in your HOME directory under the path `~/.codewind/{version}`.
 
-1.  Open a new terminal window on your local workstation
-2.  Navigate to your home directory and to the Codewind cli:
+1.  Open a new terminal window on your local workstation.
+2.  Navigate to your home directory, and then to the Codewind cli:
 
 ```
 cd ~/.codewind/0.8.0
 ```
 
-The command to remove just the Codewind deployment is:
+To remove the Codewind deployment, enter the following command:
 
 `cwctl remove remote --namespace <namespace> --workspace <workspaceID>`
 
-Where parameters `<namespace>` and `<workspaceID>` are used to identify which Kubernetes resources to remove. If you are unsure of the `<workspaceID>`, you can find it as part of the ingress or route URL used when you created the connection in your IDE:
-
-eg:
+Where parameters `<namespace>` and `<workspaceID>` are used to identify which Kubernetes resources to remove. If you are unsure of the `<workspaceID>`, you can find it as part of the ingress or route URL used when you created the connection in your IDE, for example:
 
 ```
 https://codewind-gatekeeper-k412oms7.apps.mycluster.X.X.X.X.nip.io
                             ^^^^^^^^
 ```
 
-enter the command and review the results :
+Enter the command and review the results:
```

Comment:
I noticed we have numbered steps in lines 26 and 27, but then we stop with the numbers when we reach the command line instructions. Shouldn't lines 33 and onward also be numbered? For example, 33 would be step 3, etc.